### ISSUE TITLE
Fix Request: Kotlin false positive in AvoidImplicitlyRecompilingRegex

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#688]
Next example shows only executed once because of configuration bean function.
```
@Configuration
class ApplicationConfig {

@Bean
    fun loggingFilter() = ServerLoggingFilter(Regex("/api/.*")) // <-- false positive
}
```
